libscws/xdb.c

```diff
@@ -132,7 +135,11 @@ xdb_t xdb_open(const char *fpath, int mode)
 		return NULL;
 
 	/* try to open & check the file */
+#ifdef WIN32
+	if ((x->fd = open(fpath, mode == 'w' ? O_RDWR|O_BINARY : O_RDONLY|O_BINARY)) < 0)
```

Comment:
O_BINARY: the 'binary' mode flag on Windows, this flag only exist in Windows platform.
Ref : https://msdn.microsoft.com/en-us/library/tw4k6df8.aspx